.travis.yml

```diff
@@ -32,3 +32,8 @@ script:
   # push docker images to registry if master branch:
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master"
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master" -e IMAGE_PREFIX="docker.io/avenga"
+  - > 
+    if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then 
+    make test-all build-all push-all -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master";
```

Comment:
The parameter `-e IMAGE_PREFIX="docker.io/7val` should be added here. When we change all ocurrences from 7val to avenga the images would no longer be pushed to the 7val namespace.

Reply:
Yes, I had this already in mind. This will be added to #92.  Thanks for the hint.